AppCUI/src/Utils/IniObject.cpp

```diff
@@ -457,7 +457,21 @@ IniValue IniSection::GetValue(std::string_view keyName)
     // all good -> value exists
     return IniValue(&value->second);
 }
+std::vector<IniValue> IniSection::GetValues()
+{
+    CHECK(Data, std::vector<IniValue>(), "Section was not initialized");
+
+    std::vector<IniValue> res;
+    AppCUI::Ini::Section* sect = ((AppCUI::Ini::Section*) Data);
```

Comment:
Auto daca e explicit declarat in dreapta.

Reply:
fixed